src/app/thrift-services/utils/create-next-id.ts

```diff
@@ -0,0 +1,3 @@
+export function createNextId(ids: number[]) {
+    return Math.max(...ids) + 1;
```

Comment:
Do you handling numeric overflow somehow?

Reply:
we have Int64, but it will also be possible to use deleted ids

Reply:
but yes, here we will rest on MAX_SAFE_INTEGER

Reply:
added todo